stock_return_request/models/stock_move.py

```diff
@@ -11,17 +11,8 @@ class StockMove(models.Model):
         string="Returnable Quantity",
         compute="_compute_qty_returnable",
         readonly=True,
-        store=True,
-        recursive=True,
     )
 
-    @api.depends(
```

Comment:
Why removing `depends`? Although being non stored, the depends indicate when to mark the result as dirty if cached.